packages/sdks-tests/src/e2e-tests/tracking.spec.ts

```diff
@@ -30,7 +30,7 @@ test.describe('Tracking', () => {
     }) => {
       // TO-DO: figure out why React gen1 fails this test,
       // track is not called in a published content if builder.canTrack = false
-      test.fail(packageName === 'gen1-react');
+      test.fail(packageName === 'gen1-react' || packageName === 'gen1-next-15');
```

Comment:
not sure why this is failing, could be the same reason as gen1-react but need to check

Reply:
@samijaber I think we might need to re-check this for gen1, I can see pages with symbols not respecting canTrack. Could this be because of some recent updates?